libs/wire-api/src/Wire/API/Conversation.hs

```diff
@@ -99,11 +99,15 @@ import Wire.API.Conversation.Role (RoleName, roleNameWireAdmin)
 data Conversation = Conversation
   { cnvId :: ConvId,
     cnvType :: ConvType,
+    -- FUTUREWORK: Make this a qualified user ID. This will break
+    -- backwards-compatibility with clients.
```

Comment:
Why should this break backwards-compatibility? This is an output-only type, if I'm not mistaken, so we can simply keep producing the unqualified creator id alongside the qualified one for backwards compatibility, like we are already doing for a few other types (e.g. `Event`).

Reply:
Right, I updated the comment accordingly.